tests/addresslib/parser_mailbox_test.py

```diff
@@ -520,14 +515,10 @@ def test_local_part():
     run_mailbox_test('  steve@apple.com', 'steve@apple.com')
 
     # fail dot-atom
-    run_mailbox_test('steve @apple.com', None)
-    run_mailbox_test(' steve @apple.com', None)
     run_mailbox_test(', steve@apple.com', None)
     run_mailbox_test(';;steve@apple.com', None)
     run_mailbox_test('"steve@apple.com', None)
     run_mailbox_test('steve"@apple.com', None)
-    run_mailbox_test('steve jobs @apple.com', None)
-    run_mailbox_test(' steve jobs @apple.com', None)
```

Comment:
Valid address because we allow `Foo foo@bar.com` without `<>`. Equivalent to `steve jobs@apple.com`.